README.md

```diff
@@ -460,6 +472,16 @@ Expects the following parameters:
 
 Returns [`Promise<HTTP.Body>`](#body)
 
+#### <a name="getDbConfigProperties">`db.options.getDbConfigProperties(conn)`</a>
```

Comment:
This is kind of verbose and redundant. Can we rename it to something like `db.options.getConfig` or `db.options.getConfigProperties`?

Reply:
agreed. what do you think about `getAvailable`?

Reply:
I'd lean towards it being somewhat analogous to the method syntax within stardog itself, but that also makes sense, or even something like `getDefaults` which is a little bit more descriptive and explicit in that includes the default values